src/reach_study.cpp

```diff
@@ -343,7 +343,11 @@ void runReachStudy(const YAML::Node& config, const std::string& config_name, con
 
   const boost::filesystem::path db_file = results_dir / config_name / "reach.db.xml";
 
-  if (boost::filesystem::exists(db_file))
+  YAML::Emitter emitter;
+  emitter << opt_config;
+  logger->print(emitter.c_str());
```

Comment:
```suggestion
```

This appears to be for debugging, so let's remove this

Reply:
Oh yes sorry, already removed